code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -147,6 +147,10 @@
 /datum/reagent/water/expose_turf(turf/open/T, reac_volume)
 	if(!istype(T))
 		return
+
+	// This is almost certainly not the correct usage of signals and very much should be generalized.
+	SEND_SIGNAL(T, COMSIG_ACID_DILUTE, 0, reac_volume) // TODO: Make a robust pH system so that this can be handled generally.
```

Comment:
Add this to the base: `/datum/reagent/proc/expose_turf(turf/T, volume)`
And then call parent here.

Reply:
That will extend the effect to all reagents. Not just water as is the current behavior. Is this acceptable?

Reply:
Time for scope creep.

Reply:
I have applied scope creep. I think it worked.